src/components/Header.jsx

```diff
@@ -87,6 +87,11 @@ const MbubeHeader = ({ siteTitle }) => (
                     to: '/bli-speider/',
                     onClick: () => navigate('/bli-speider/'),
                   },
+                  {
```

Comment:
Foreslår at "Bilder" kommer før "Bli speider". 

Mener det er en fordel at "Bli speider" fungerer som en slags diskret call-to-action, hvis den står for seg selv.